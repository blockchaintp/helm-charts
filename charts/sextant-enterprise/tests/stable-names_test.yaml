suite: sextant-enterprise.test for stable names
templates:
  - charts/sextant/templates/serviceaccount.yaml
  - charts/sextant/templates/statefulset.yaml
  - charts/sextant/templates/service.yaml
release:
  name: test
  namespace: testns
tests:
  - it: check for stable entity names (services, statefulsets, serviceaccounts)
    asserts:
      - matchSnapshot:
          path: metadata.name
  - it: check for stable PVC names
    template: charts/sextant/templates/statefulset.yaml
    set:
      sextant:
        postgres:
          persistence:
            enabled: true
    asserts:
      - matchSnapshot:
          path: spec.volumeClaimTemplates[0].metadata.name
