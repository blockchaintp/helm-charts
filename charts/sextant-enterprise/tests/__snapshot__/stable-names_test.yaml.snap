check for stable PVC names:
  1: |
    pgdata
check for stable entity names (services, statefulsets, serviceaccounts):
  1: |
    test-sextant-enterprise
  2: |
    test-sextant-enterprise
  3: |
    test-sextant-enterprise
